### ISSUE TITLE
[FEATURE] Add validation of enabled GCP APIs

### ISSUE THREAD

**Issue by blumem** [state-of-the-art/BlendNet#20]
### Is your feature request related to a problem? Please describe.
To catch insufficient setups early, the addon could validated the list of required APIs against the list of enabled APIs for the current project

### Describe the solution you'd like
Right now the documentation leaves the GCP setup part to the user. There is a risk that novice cloud users will fail to complete the setup properly. The addon could do one setup to validate the list of enabled APIs during addon activation against a known list of minimal required PAIs

### Describe alternatives you've considered
Just add a more detailed setup guide for the cloud piece

### Additional context

`gcloud services list --enabled
NAME                              TITLE
...
cloudapis.googleapis.com          Google Cloud APIs
compute.googleapis.com            Compute Engine API
datastore.googleapis.com          Cloud Datastore API
deploymentmanager.googleapis.com  Cloud Deployment Manager V2 API
storage-api.googleapis.com        Google Cloud Storage JSON API
storage-component.googleapis.com  Cloud Storage
...`

**Comment by rabits** [comment]
Right, first of all need to prepare documentation what is needed to be enabled - and yeah, it's better to guide the user by warnings about "what's wrong with the project setup".
